### PR TITLE
As an American, I had to correct the spelling of 'independent'

### PR DESCRIPTION
**Pull request by NVentimiglia**
Spell checked some comments.

### CODE REVIEW

File: src/Microsoft.AspNetCore.Server.Kestrel/Internal/Infrastructure/MemoryPoolBlock.cs [hunk 1]
```diff
@@ -65,7 +65,7 @@ unsafe protected MemoryPoolBlock(IntPtr dataArrayPtr)
 
         /// <summary>
         /// Reference to the next block of data when the overall "active" bytes spans multiple blocks. At the point when the block is
-        /// leased Next is guaranteed to be null. Start, End, and Next are used together in order to create a linked-list of discontiguous 
+        /// leased Next is guaranteed to be null. Start, End, and Next are used together in order to create a linked-list of discontinuous
```
Comment (benaadams) [inline src/Microsoft.AspNetCore.Server.Kestrel/Internal/Infrastructure/MemoryPoolBlock.cs]: This is a word with a different meaning

**Comment by cesarbs** [comment]
:us: :ship: :us:

**Comment by NVentimiglia** [comment]
I have accepted the cla @dnfclas

**Comment by Eilon** [comment]
@NVentimiglia for the CLA did you get a confirmation email from DocuSign?

**Comment by NVentimiglia** [comment]
I have accepted the cla @dnfclas

**Comment by Eilon** [comment]
@NVentimiglia thanks!

**Comment by halter73** [comment]
:shipit:

**Pull request merged**